pkg/checks/external/ssl_util/main.go

```diff
@@ -17,67 +19,148 @@ import (
 
 var TimeoutSeconds = 10
 
-// CertHandshake performs a basic TLS/SSL handshake on the specified host/port, returning any errors
-func CertHandshake(host, port string) error {
-	log.Info("Testing SSL handshake on host ", host, " over port ", port)
+// FileUploaded returns 'true' if the pem formatted cert has been mounted at /etc/ssl/selfsign/certificate.crt
+func FileUploaded() bool {
```

Comment:
some of these functions don't look like they're used outside of this file -- you can lowercase them `fileUploaded()`, `CertFromHost()`, `PoolFromHost()`, etc.

Reply:
Lowercased all functions not used outside `ssl_util` package.